build/Dockerfile

```diff
@@ -1,16 +1,9 @@
-FROM node:16.13.0-buster
```

Comment:
I hope there is no particular reason using buster

Reply:
while we are at it, could you update this to Node v18 instead?

Reply:
@mcollina sure thing! done